### PR TITLE
Add global setting for auto cleanup systen generated snapshot

### PR DESCRIPTION
**Pull request by boknowswiki**
https://github.com/longhorn/longhorn/issues/1669

Signed-off-by: Bo Tao <bo.tao@rancher.com>

### CODE REVIEW

File: types/setting.go [hunk 1]
```diff
@@ -503,6 +506,16 @@ var (
 		ReadOnly:    false,
 		Default:     "true",
 	}
+
+	SettingDefinitionAutoCleanupSystemGeneratedSnapshot = SettingDefinition{
+		DisplayName: "Auto Cleanup System Generated Snapshot",
```
Comment (yasker) [inline types/setting.go:511]: `Automatically Cleanup ...`

**Review by yasker — CHANGES_REQUESTED** [review]
Also need to add to `deploy/install/01-prerequisite/04-default-setting.yaml`

**Comment by boknowswiki** [comment]
> Also need to add to `deploy/install/01-prerequisite/04-default-setting.yaml`

Oh, yes. Sorry about that. I will add it. Thanks for reminder.

File: types/setting.go (hunk 1 above)
Comment (boknowswiki) [reply types/setting.go]: Will fix it, thanks.

**Review by yasker — APPROVED** [review]

**Pull request merged**